### PR TITLE
create stub file for KC

### PR DESCRIPTION
**Pull request by icatchpole**
@icatchpole What do you think ?

This pull request adds a file for Kansas City. I would like to add places to each and things to do.

### CODE REVIEW

File: icatchpole-kc.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Anchorage AK
+
+## Places to Eat
+- Glacier Brew House
+- Simon and Seaforts
+- Benihana
+
+## Things to Do
```
Comment (fabgonzalez) [inline icatchpole-kc.md:8]: @icatchpole What time of the year would be the best to visit Anchorage?

File: icatchpole-kc.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Anchorage AK
+
+## Places to Eat
+- Glacier Brew House
+- Simon and Seaforts
+- Benihana
```
Comment (fabgonzalez) [inline icatchpole-kc.md:6]: What type of food is this?

**Review by fabgonzalez — COMMENTED** [review]
@icatchpole It sounds pretty cool. I always wanted to visit Alaska. I wonder what is the best time to do it :sunglasses:

File: icatchpole-kc.md (hunk 1 above)
Comment (icatchpole) [reply icatchpole-kc.md:8]: During the summer months May to September, unless you like winter activities.

File: icatchpole-kc.md (hunk 2 above)
Comment (icatchpole) [reply icatchpole-kc.md:6]: General American food.  More upscale.

**Pull request merged**